app/views/contexts/calculated_summary_context.py

```diff
@@ -87,12 +85,12 @@ def build_view_context_for_calculated_summary(
         }
 
     def _build_calculated_summary_section(
-        self, rendered_block: ImmutableDict, current_location: Location
+        self, rendered_block: Dict, current_location: Location
```

Comment:
We need to follow our [type hints guide in runner](https://github.com/ONSdigital/eq-questionnaire-runner/blob/master/doc/python-type-hinting.md):
"For standard collections, use lower case names e.g `dict` rather than `Dict`"
also "Specify at least the top-level type parameters for all generic types" so for example `dict[str, int]` instead of  just `dict`.
Quite a few more of these to fix in this PR.

Reply:
Agree with @petechd's comment above, but in this instance, not sure why this had changed from ImmutableDict in the first place?

Reply:
Right it should be ImmutableDict 👍